From daeddb16ebf1a80c1d587b35af24a30a44aceff6 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Fri, 25 Sep 2020 11:07:19 +0800
Subject: [PATCH] ARM: rockchip_defconfig: Enable CONFIG_DM_BOW

https://android.googlesource.com/kernel/configs
commit 8c7922ea12ed ("android-4.14, android-4.19, android-5.4: Disable KPROBES for R")

+CONFIG_DM_BOW=y

commit c85479e4c183 ("Add CONFIG_DM_BOW to recommended")
If you are using Q on ext4 userdata, you need this feature.

Signed-off-by: Tao Huang <huangtao@rock-chips.com>
Change-Id: I2b1f942a775656435104984c2e85095e78c91435
---
 arch/arm/configs/rockchip_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/rockchip_defconfig b/arch/arm/configs/rockchip_defconfig
index f6f22bdbdad9..bce7a29f4175 100644
--- a/arch/arm/configs/rockchip_defconfig
+++ b/arch/arm/configs/rockchip_defconfig
@@ -254,6 +254,7 @@ CONFIG_DM_UEVENT=y
 CONFIG_DM_VERITY=y
 CONFIG_DM_VERITY_AVB=y
 CONFIG_DM_VERITY_FEC=y
+CONFIG_DM_BOW=y
 CONFIG_DM_ANDROID_VERITY_AT_MOST_ONCE_DEFAULT_ENABLED=y
 CONFIG_NETDEVICES=y
 CONFIG_DUMMY=y
-- 
2.35.3

